download_script.py

```diff
@@ -0,0 +1,50 @@
+import requests
+import argparse
+
+# help function for better understanding of the scripts functionality
+
+description = """  This script is used to download all the pickle/dataset files needed to train, evaluate and predict using the model.
+                    Also downloads model tokenizers used for preprocessing the datasets before feeding them to the model \n
+
+                   The script assumes to be run at spambrainz_ml top directory \n
+
+                   The files if present are overwritten by the script, kindly be careful"""
```

Comment:
1. Full stops `.``are missing at the end of each sentence.
2. `\n`s and line breaks are ignored by the default formatter. Either remove them or use another [formatter](https://docs.python.org/3/library/argparse.html#formatter-class) such as `RawDescriptionHelpFormatter`.